Sources/ArgumentParser/Usage/HelpGenerator.swift

```diff
@@ -90,7 +143,6 @@ internal struct HelpGenerator {
   var abstract: String
   var usage: String
   var sections: [Section]
-  var discussionSections: [DiscussionSection]
```

Comment:
question: What happened to the discussion sections? I'm guessing they were vestigial.

Reply:
Ah, I think I accidentally deleted them. I didn't see them being used anywhere, so I may be able to remove it entirely. Will investigate as I continue this PR.